Formula/postgresql.rb

```diff
@@ -138,35 +138,12 @@ def caveats
     EOS
   end
 
-  plist_options manual: "pg_ctl -D #{HOMEBREW_PREFIX}/var/postgres start"
-
-  def plist
-    <<~EOS
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
-      <dict>
-        <key>KeepAlive</key>
-        <true/>
-        <key>Label</key>
-        <string>#{plist_name}</string>
-        <key>ProgramArguments</key>
-        <array>
-          <string>#{opt_bin}/postgres</string>
-          <string>-D</string>
-          <string>#{postgresql_datadir}</string>
-        </array>
-        <key>RunAtLoad</key>
-        <true/>
-        <key>WorkingDirectory</key>
-        <string>#{HOMEBREW_PREFIX}</string>
-        <key>StandardOutPath</key>
-        <string>#{postgresql_log_path}</string>
-        <key>StandardErrorPath</key>
-        <string>#{postgresql_log_path}</string>
-      </dict>
-      </plist>
-    EOS
+  service do
+    run [opt_bin/"postgres", "-D", @formula.postgresql_datadir]
```

Comment:
originally, i didn't use them because i could tell it was unintended usage. however, for the versioned formula this caused repeating the formula name three times:

```diff
diff --git a/Formula/postgresql@12.rb b/Formula/postgresql@12.rb
index e779dd8056..743a215480 100644
--- a/Formula/postgresql@12.rb
+++ b/Formula/postgresql@12.rb
@@ -173,10 +173,10 @@ class PostgresqlAT12 < Formula
   end
 
   service do
-    run [opt_bin/"postgres", "-D", @formula.postgresql_datadir]
+    run [opt_bin/"postgres", "-D", var/"postgres@12"]
     keep_alive true
-    log_path @formula.postgresql_log_path
-    error_log_path @formula.postgresql_log_path
+    log_path var/"log/postgres@12.log"
+    error_log_path var/"log/postgres@12.log"
     working_dir HOMEBREW_PREFIX
   end
```

is that ok?

Reply:
Yeah, that's fine. I think it's pretty easy to read.

Reply:
You forgot to fix this one.